lib/HTTP/Action.js

```diff
@@ -10,12 +10,12 @@ class HTTPAction extends Action {
     this.httpConfig = httpConfig;
   }
 
-  async request(q, method, headers, body) {
+  async request(q, method = 'POST', headers, body) {
```

Comment:
I think a param with default followed by params without default isn't great. What about putting method, headers and body in a simple object?